client/src/components/SignUp.tsx

```
@@ -23,232 +1,220 @@
           />
-          <button
-            type="button"
-            onClick={() => setShowConfirmPassword(!showConfirmPassword)}
-            className="absolute inset-y-0 right-2 flex items-center text-gray-600"
-          >
-            {showConfirmPassword ? <EyeOff size={20} /> : <Eye size={20} />}
-          </button>
+          {errors.confirmPassword && (
+            <span className="text-red-500">
+              {errors.confirmPassword &&
+                errors.confirmPassword.message &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)